### ISSUE TITLE
What do I do?

### ISSUE THREAD

**Issue by Anime-champian** [itchio/itch#3123]
Something went wrong

itch-setup, v1.26.0, built on Apr 21 2021 @ 20:48:11, ref 48f97b3e7b0b065a2478811b8d0ebcae414845fd

Get "https://broth.itch.ovh/itch/windows-amd64/LATEST": dial tcp: lookup broth.itch.ovh: no such host
While performing GET request to https://broth.itch.ovh/itch/windows-amd64/LATEST
while getting latest version
Error during warm-up

**Comment by lexi-the-cute** [comment]
yep. the dns server for it is down atm. you may have to manually upload a copy of the butler binary to your CI. if it's still down by the time Monday rolls around, i'll have to upload a copy somewhere to access via Github Actions.

https://itchio.itch.io/butler is where an official download can be found that is not down (although not easily automatable)

**Comment by lexi-the-cute** [comment]
so, there's 5 separate issues open for the same error. i'm not part of the Itch team, so i can't comment on if they read these issues. would like to report that we should focus on a single issue. the original one being https://github.com/itchio/itch/issues/3122

**Comment by leafo** [comment]
Please try again, we had some downtime with our legacy update server that was still hard-coded in a few places.

That said, we've since pushed out patched versions of the app and related software. Please make sure you update to the latest version to ensure you don't have any issues accessing the app in the future: https://itch.io/t/4351835/app-and-butler-updates-may-require-manual-intervention
